content/docs/command-reference/exp/remove.md

```diff
@@ -56,3 +57,43 @@ $ dvc exp list
 master:
         exp-e6c97
 ```
+
+The same applies to queued experiments but they won't have a name to give to
+`dvc exp remove` yet unless you specify one. Alternatively, you can use their ID
+instead (shown when queued and by `dvc exp show`).
+
+Let's queue a few experiments in different ways and then delete some of them:
+
+```dvc
+$ dvc exp run --queue -S train.min_split=64
+Queued experiment 'e41d5b4' for future execution.
+$ dvc exp run --queue -S train.min_split=32 --name split32
+Queued experiment '5751540' for future execution.
+$ dvc exp run --queue -S train.min_split=16 --name split16
+Queued experiment '8de9a6c' for future execution.
+
+$ dvc exp e41d5b4 split16
+
+$ dvc exp show --include-params=train.min_split --no-pager
+```
+
+```table
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ Experiment            ┃ Created      ┃ State  ┃ avg_prec ┃ roc_auc ┃ train.min_split ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace             │ -            │ -      │  0.57553 │ 0.94652 │ 2               │
+│ master                │ Aug 02, 2021 │ -      │  0.53252 │  0.9107 │ 2               │
+│ └── 5751540 [split32] │ 04:57 PM     │ Queued │        - │       - │ 32              │
+└───────────────────────┴──────────────┴────────┴──────────┴─────────┴─────────────────┘
+```
+
+```dvc
+$ dvc exp remove --queue
+$ dvc exp show --include-params=train.min_split --no-pager
+┏━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ Experiment ┃ Created      ┃ avg_prec ┃ roc_auc ┃ train.min_split ┃
+┡━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace  │ -            │  0.57553 │ 0.94652 │ 2               │
+│ master     │ Aug 02, 2021 │  0.53252 │  0.9107 │ 2               │
+└────────────┴──────────────┴──────────┴─────────┴─────────────────┘
+```
```

Comment:
Let's delete this and create an **Example: remove all experiments** in https://github.com/iterative/dvc.org/pull/2722 instead, and throw this in there?